code/modules/clothing/gloves/color.dm

```diff
@@ -12,6 +12,14 @@
 	custom_price = PAYCHECK_MEDIUM * 10
 	custom_premium_price = PAYCHECK_COMMAND * 6
 	cut_type = /obj/item/clothing/gloves/cut
+	clothing_traits = list(TRAIT_CHUNKYFINGERS)
+
+/obj/item/clothing/gloves/color/yellow/chief_engineer
```

Comment:
I think this should be its own subtype under `color`, since most of insulated gloves' behaviors either don't apply here (these aren't in vendors) or shouldn't (cut gloves won't look the same).